app/views/admin/recovery_displays/_form.html.haml

```diff
@@ -59,12 +59,13 @@
                     #{link_to "Bike", bike_url(@bike), target: "_blank"} #{text}
                   %a.btn.btn-success#use_image_for_display{ href: '#', data: {url: url}}
                     Use first image
-                    = f.hidden_field :remote_image_url
```

Comment:
I think you need this field.

You'll need javascript to fill in the remote image url there